source/simulator/solver.cc

```diff
@@ -458,7 +483,15 @@ namespace aspect
             SolverControl solver_control(5000, 1e-6 * src.l2_norm(), false, true);
             SolverCG<TrilinosWrappers::MPI::Vector> solver(solver_control);
             //Solve with Schur Complement approximation
-            solver.solve(mp_matrix,
+            auto Op_A=LinearOperator<TrilinosWrappers::MPI::Vector>(system_matrix.block(0,0));
+            auto Op_BT = LinearOperator<TrilinosWrappers::MPI::Vector>(system_matrix.block(0,1));
```

Comment:
Let's get this neatly aligned so that it's easoer to read.
```suggestion
            auto Op_A   = LinearOperator<TrilinosWrappers::MPI::Vector>(system_matrix.block(0,0));
            auto Op_BT = LinearOperator<TrilinosWrappers::MPI::Vector>(system_matrix.block(0,1));
```